core/services/relay/evm/functions/offchain_config_digester.go

```diff
@@ -0,0 +1,134 @@
+package functions
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/smartcontractkit/libocr/gethwrappers2/exposedocr2aggregator"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+var (
+	_                     types.OffchainConfigDigester = FunctionsOffchainConfigDigester{}
+	configDigestArgs                                   = makeConfigDigestArgs()
+	FunctionsDigestPrefix                              = types.ConfigDigestPrefixEVM
+	ThresholdDigestPrefix                              = types.ConfigDigestPrefix(7)
+	S4DigestPrefix                                     = types.ConfigDigestPrefix(8)
+)
+
+type FunctionsOffchainConfigDigester struct {
```

Comment:
How about embedding evmutil.EVMOffchainConfigDigester, calling its methods and then overriding the prefix afterwards?

Simplified example: https://go.dev/play/p/0RDoywXY-8S

Digester seems to be a perfect candidate for that approach. Not so sure about Poller, though...
Not sure if that make